src/main/java/io/github/railroad/Railroad.java

```diff
@@ -171,6 +170,10 @@ public void start(Stage primaryStage) {
             HTTP_CLIENT.dispatcher().executorService().shutdown();
             HTTP_CLIENT.connectionPool().evictAll();
         });
+        } catch (Exception e) {
```

Comment:
Possibly keep this here, otherwise it may unexpectedly close on the end user, and this could be used for simple debugging?